release/models/virtual-devices/openconfig-virtual-ports.yang

```diff
@@ -468,5 +475,36 @@ module openconfig-virtual-ports {
     }
   }
 
+  notification oper-status-change {
+    if-feature "oper-status-change-notification";
+    description
+      "A virtual port operational state notification.";
+
+    leaf global-resource-id {
+      type uint32;
+      mandatory true;
+      description
+        "Identifies virtual port by global unique id.";
+    }
+
+    leaf oper-status {
+      type enumeration {
+        enum UP {
```

Comment:
Using `UP` and `DOWN` seems better than using the `LINK_UP` and `LINK DOWN` as used in the oper-status above. This also aligns better with other models such as the oc:interfaces model where oper-status is indicated by just UP and DOWN enums.